static/css/egi.css

```diff
@@ -0,0 +1,142 @@
+.features {
```

Comment:
It would be good to have some header/comments explaining why we have all this customised.

Reply:
Added